### PR TITLE
INREL-3969 Finalize Lexicon

### PR DESCRIPTION
**Pull request by danschalow**
## [INREL-3969](https://jira.burda.com/browse/INREL-3969) 
Styled Lexicon page
 - [elle:PR#162](https://github.com/BurdaMagazinOrg/elle-web/pull/162)

## How to test:
Please provide a brief step-by-step instruction.
- ensure shared_config is checked out to the same branch
- https://feature1.elle.de/modelexikon

## Checklist:

- [x] I have verified that the code works
- [x] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [ ] I have [left the code in a better state](http://programmer.97things.oreilly.com/wiki/index.php/The_Boy_Scout_Rule)
- [ ] I have documented the changes (where applicable)

### CODE REVIEW

File: sass/components/lexicon/_lexicon.scss [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    }
+
+    .teaser__copy {
+      @include text-helper($font-content);
+    }
+
+    @media (max-width: $screen-xs-max) {
+      .teaser__title {
+        @include text-helper-size($font-content-bold, $title-size-xs);
+        margin-top: 8px;
```
Comment (saviomuc) [inline sass/components/lexicon/_lexicon.scss]: I think we have variables for that. Please check `_variables.base.gaps.scss`

File: sass/components/lexicon/_lexicon.scss [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+      }
+
+      .teaser__copy {
+        @include text-helper-size($font-content, $font-size-medium, 20px);
```
Comment (saviomuc) [inline sass/components/lexicon/_lexicon.scss]: Please also avoid magic numbers here. You should consider move that to a specific var.

File: sass/components/lexicon/_lexicon.scss [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+        margin-top: 6px;
```
Comment (saviomuc) [inline sass/components/lexicon/_lexicon.scss]: Magic number =)

File: sass/components/lexicon/_lexicon.scss [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+      }
+    }
+
+    @media (min-width: $screen-sm-min) {
+      .teaser__title {
+        @include text-helper-size($font-content-bold, $title-size-md);
+        margin-top: 8px;
```
Comment (saviomuc) [inline sass/components/lexicon/_lexicon.scss]: Same here

File: sass/components/lexicon/_lexicon.scss [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+      li {
+        flex-grow: 1;
+        flex-shrink: 0;
+        align-self: auto;
+        margin-top: 10px;
+        box-sizing: border-box;
+
+        .teaser__img-container {
+          height: 0;
+          padding-bottom: 56.2%;
```
Comment (saviomuc) [inline sass/components/lexicon/_lexicon.scss]: There should be a aspect ratio mixin for that.

**Review by saviomuc — CHANGES_REQUESTED** [review]
Looks good so far. Some small nitpicks and please check the templates (where are they from)

**Comment by danschalow** [comment]
- Replaced all magic numbers

File: sass/components/lexicon/_lexicon.scss [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+
+    @media (min-width: $screen-sm-min) {
+      .teaser__title {
+        @include text-helper-size($font-content-bold, $title-size-md);
+        margin-top: $lexicon-teaser__title--margin-top__tablet;
+      }
+
+      .teaser__copy {
+        @include text-helper-size($font-content, $font-size-medium, $lexicon--line-height);
+        margin-top: 6px;
```
Comment (saviomuc) [inline sass/components/lexicon/_lexicon.scss]: Please use `$lexicon-teaser__copy--margin-top__mobile;`

File: sass/components/lexicon/_lexicon.scss [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+      .lexicon-teaser__item {
+        flex-grow: 1;
+        flex-shrink: 0;
+        align-self: auto;
+        margin-top: $lexicon-teaser--margin-top;
+        box-sizing: border-box;
+
+        .teaser__img-container {
+          @include aspect-ratio(264, 148.5);
+          background: #e7e7e7;
```
Comment (saviomuc) [inline sass/components/lexicon/_lexicon.scss]: Please move to vars.

**Review by saviomuc — CHANGES_REQUESTED** [review]

**Review by saviomuc — APPROVED** [review]
LGTM :shipit:

**Pull request merged**